ambari-server/src/main/java/org/apache/ambari/server/api/AmbariViewErrorHandlerProxy.java

```diff
@@ -51,14 +52,18 @@ public AmbariViewErrorHandlerProxy(ErrorHandler webAppErrorHandler, AmbariErrorH
 
   @Override
   public void handle(String target, Request baseRequest, HttpServletRequest request, HttpServletResponse response) throws IOException {
-
+   try{
     if (isInternalError(request, response)) {
       //invoke the ambari error handler
       ambariErrorHandler.handle(target, baseRequest, request, response);
     } else {
       //invoke the original errorhandler
       webAppErrorHandler.handle(target, baseRequest, request, response);
     }
+   }
+   catch(IOException | ServletException e){
```

Comment:
I thought @brahmareddybattula 's comment was to catch ServletException. Perhaps also encaupsulate the exception with an IOException and then throw it out?